src/net.cpp

```diff
@@ -1745,7 +1745,8 @@ void ThreadOpenConnections()
         int nOutbound = 0;
         int nThinBlockCapable = 0;
         set<vector<unsigned char> > setConnected;
-        CNode *ptemp = nullptr;
+        CNode *ptemp1 = nullptr;
```

Comment:
ptemp1 and ptemp2 are pretty generic variable names, making this hard to follow :)